README.md

```diff
@@ -346,100 +337,80 @@ If the column that best matches your system is red, then maybe things aren't
 just broken for you. If it's green, then you can click the "stdio" links in the
 latest build to see what commands the build bots run, and what the output was.
 
-# Some useful environment variables
```

Comment:
I think the paragraph above about how to interpret the buildbot status is out of date. The buildbot UI changed at some point.

Reply:
Updated! It wasn't _too_ bad, but I made it clearer, I hope.